apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -281,6 +281,28 @@ public int Port
             set => SetInt32WithValidation(nameof(Port), 1, 65535, value);
         }
 
+        /// <summary>
+        /// The universe domain to connect to, or null to use the default universe domain <see cref="DefaultDomain"/>.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// <see cref="UniverseDomain"/> is used to build the endpoint to connect to, unless <see cref="EndPoint"/> or <see cref="Host"/>
+        /// is explicitly set, in which case <see cref="EndPoint"/> will be used without further modification.
+        /// </para>
+        /// <para>
+        /// If default credentials or one of <see cref="GoogleCredential"/> or <see cref="CredentialFile"/>
+        /// is used, <see cref="GoogleCredential.GetUniverseDomain"/> should be:
+        /// <list type="bullet">
+        /// <item>The same as <see cref="UniverseDomain"/> if <see cref="UniverseDomain"/> has been set otherwise should be same as <see cref="DefaultDomain"/></item>
+        /// </list>
```

Comment:
I don't think we need a list if it's going to have a single item.

```suggestion
        /// is used, <see cref="GoogleCredential.GetUniverseDomain"/> should be
        /// the same as <see cref="UniverseDomain"/> if <see cref="UniverseDomain"/> has been set otherwise should be same as <see cref="DefaultDomain"/>
```